activerecord/lib/active_record/associations/join_dependency/join_association.rb

```diff
@@ -28,9 +28,7 @@ class JoinAssociation < JoinPart # :nodoc:
         delegate :table, :table_name, :to => :parent, :prefix => :parent
         delegate :alias_tracker, :to => :join_dependency
 
-        #TODO - Bogdan - this is only temporary until figuring it out on the missing / undefined parent_table_name attribute
-        #alias :alias_suffix :parent_table_name
-        alias :alias_suffix :table_name
+        alias :alias_suffix :parent_table_name
```

Comment:
In this case I wouldn't worry about it because this change seems like a leftover from when we were moving from Rails 2 to 3, and it looks like if reverting it was problematic, then web would break horribly. in this case, the code looks totally equivalent, and I'm betting that Bogdan was shooting in the dark and just trying different adjustments to fix a problem, found the source of the problem elsewhere, then forgot to change this back.